docs/services/camera_sim.rst

```diff
@@ -1,14 +1,59 @@
 Camera Simulator
 ================
+This service operates a simulated camera. This service is meant to mimic, but does not actually
+control, a hardware camera.
+
+When applicable, all services have a corresponding simulated service to be able to test control
+software before commanding actual hardware devices.
 
 Configuration
 -------------
+.. code-block:: YAML
+
+    camera:
+      service_type: zwo_camera
+      simulated_service_type: camera_sim
+      interface: hicat_camera
```

Comment:
I think better just `camera` here since that one actually lives inside of catkit2 itself.